codegen/datamodelgen.py

```diff
@@ -223,6 +239,9 @@ def _write_static_info(self, name: str, info: Any, f: FileIO, level: int = 0):
         singletons = sorted(info.singletons)
         parameters = sorted(info.parameters)
         commands = sorted(info.commands)
+
+        if int(self.version) >= 241:
+            queries = sorted(info.queries)
```

Comment:
Here also the queries can be an empty list otherwise, the version checks won't be required at other places of this method.

Reply:
Done. Thank you.